Geofencing/app/src/main/java/com/google/android/gms/location/sample/geofencing/Constants.java

```diff
@@ -42,8 +43,8 @@ private Constants() {
     /**
      * For this sample, geofences expire after twelve hours.
      */
-    static final long GEOFENCE_EXPIRATION_IN_MILLISECONDS =
-            GEOFENCE_EXPIRATION_IN_HOURS * 60 * 60 * 1000;
+    static final long GEOFENCE_EXPIRATION_IN_MILLISECONDS = Geofence.NEVER_EXPIRE;
+            //GEOFENCE_EXPIRATION_IN_HOURS * 60 * 60 * 1000;
```

Comment:
Can you explain why you have moved this to never expire?  If this change is intentional we need to remove the commented code and update the JavaDoc comment above.

Reply:
The idea was to showcase that the Geofences will trigger even if the app has been killed. If the OS decides to kill the app, the notification should still trigger after entering/leaving the Geofence. One can manually kill the app and verify that the behavior is correct.

I suggest reverting to the original 12 hour expiration. I will go ahead and push that change.

Thank you,

Reply:
@samtstern done, the 12 hour expiration is restored